packages/web/components/TechnologyForm/MapAndAttachments/ImagesPreview.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { FiTrash2, FiImage } from 'react-icons/fi';
+
+import { IconRow, Media, Button, RadioWrapper } from './styles';
+
+const ImagesPreview = ({ previewedImgFiles, value, onChange, deleteAttachment }) => {
+	if (!previewedImgFiles?.length) {
+		return null;
+	}
+
+	return previewedImgFiles.map((element, index) => (
+		<IconRow key={element.url} alignItems="flex-start">
+			<RadioWrapper>
+				<input
+					id={element.url}
+					type="radio"
+					name="thumbnail_id"
+					value={element.id}
+					defaultChecked={value === element.id || !index}
+					onChange={onChange}
+				/>
+				{/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+				<label htmlFor={element.url}>
+					<FiImage fontSize="1.6rem" />
+					Usar como capa
+				</label>
```

Comment:
Did you see this comment? https://github.com/ufersa/plataforma-sabia/pull/558#discussion_r512611697 🤔

Reply:
Yes bro no way to achieve this style doing that. Ive explained it in the second comment after that one youve mentioned

Reply:
Sorry bro 👀

Reply:
No problem hahaha take it easy